packages/cli/src/cmds/record/recordContext.ts

```diff
@@ -7,7 +7,6 @@ import {
 } from './configuration';
 
 export type RecordProcessResult = {
-  env: Record<string, string>;
```

Comment:
This changes originally included the process environment in the telemetry events, but this seemed risky. Let me know if you think I should put it back.